File: BOJ/1929.js
const input = require('fs').readFileSync('/dev/stdin').toString().split(' ').filter(Boolean);
const min = parseInt(input[0], 10);
const max = parseInt(input[1], 10);
let str = '';
function isPrime(N) {
    if (N < 2) return false;
    for (let i = 2; i*i <= N; i += 1) {
        if (N % i === 0) return false;
    }
    return true;
}
for (let i = min; i <= max; i += 1) {
    if (isPrime(i)) str += i + '\n'; 
}
console.log(str);

File: BOJ/1929.cpp
#include <stdio.h>
bool isPrime(int N) {
    if (N < 2) return false;
    for (int i = 2; i*i <= N; i += 1) {
        if (N % i == 0) return false;
    }
    return true;
}
int main() {
    int n, m;
    scanf("%d %d", &n, &m);
    for (int i = n; i <= m; i += 1) {
        if (isPrime(i)) printf("%d\n", i);
    }
}